npm/private/yaml.bzl

```diff
@@ -81,6 +81,29 @@ def _handle_CONSUME_SPACE(state, input, stack, starlark):
         # Consume any space following the {
         stack.append(_new_CONSUME_SPACE_FLOW())
 
+    elif input == "?":
+        stack.pop()
+
+        # We are at the beginning of a new complex map
```

Comment:
Is it more correct to say that we're at the beginning of a complex key, since a map can have complex and normal keys?